script.js

```diff
@@ -15,6 +15,34 @@ mobileMenuItems.addEventListener('click', () => {
   mobileMenu.classList.toggle('d-block');
 });
 
+const form = document.querySelector('.form');
+const validateEmail = document.querySelector('input[type=email]');
+const errorMessagesContainer = document.querySelector('.error-messages');
+
+const showErrors = (errors) => {
+  errors.forEach((error) => {
+    const li = document.createElement('li');
+    li.innerText = error;
+    li.classList.add('programming-language');
+    li.style.color = 'red';
+    li.style.marginBottom = '16px';
+    li.style.padding = '14px';
+    errorMessagesContainer.append(li);
+  });
+  errorMessagesContainer.classList.toggle('d-none');
+};
+
+form.addEventListener('submit', (e) => {
+  const errorMessages = [];
```

Comment:
Kindly implement an error message for  entering uppercase letters in the mail field

Reply:
Can you please elaborate a little bit more?

Reply:
when i fill the email field with capital letters the form does not give the error message as it should as the email field should accept only small letters

Reply:
https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/9_validate_form.md

In the above link, they asked us to show the error on submit that's why It'll show the error on submit and not before submit.

Reply:
It does not show an error at all

Reply:
## Kindly, check below the button

![Screenshot from 2021-07-31 02-13-47](https://user-images.githubusercontent.com/83048208/127712176-69ce1088-4482-42ed-910a-d6335f4d0d20.png)

Reply:
submit the PR and it will approved

Reply:
I've to submit for a second review? But it's still showing changes required.

Reply:
You need to submit the code review request so i can approve it

Reply:
But I already did, Kindly tell me If you're asking for a second review request?

Reply:
Yes please

Reply:
Alright, I submitted the PR.

Reply:
Issue not resolved. The email field  of the form is accepting uppercase letters without displaying error message  Kindly take some time to implement this changes requested by the previous reviewer.

Reply:
Kindly, explain where it's not working. I checked it several times applying different values both upper and lower. and It doesn't let pass the upper case value.

Reply:
### Kindly, check below the button. They asked us to create error message near to submit button and this is what I did.
![Screenshot from 2021-07-31 02-13-47](https://user-images.githubusercontent.com/83048208/127714418-3e0e702c-806d-4e37-95ec-186267f9876d.png)